### PR TITLE
added some legendary quotes

### PR DESCRIPTION
**Pull request by cherouvim**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,6 +4,14 @@ export default [
...
-  'We’ll fix it later',
+  'Looks like it’s all working fine.',
+  'We’ll fix it later.',
+  'Does it support blockchain?',
+  "Didn't we refactor last year?",
+  'Technical debt? Never heard of it.',
+  'Writing tests takes x3 time than writing actual code.',
+  'We need you to work on features. Not performance.',
+  'Please do not worry a lot about quality. This is a prototype.',
+  'Hurry up. This is just a proof of concept.',
```
Comment (Return-1) [inline src/index.js]: I think this one is similar to the one above. Otherwise its all good with me. @jmike ?

Comment (jmike) [reply src/index.js]: Agreed, @cherouvim line #15 is quite similar to line #14.
Would you agree to remove it from the PR?

File: src/index.js [hunk 2]
```diff
@@ -4,6 +4,14 @@ export default [
...
+  'Can we worry about security at a later phase?'
```
Comment (jmike) [inline src/index.js]: @cherouvim suggestion to simplify: "Can we worry about security later?"

What do you think?

**Review by jmike — CHANGES_REQUESTED** [review]
Awesome stuff. A couple of comments on my side.

File: src/index.js (hunk 1 above)
Comment (cherouvim) [reply src/index.js]: Done.

File: src/index.js (hunk 2 above)
Comment (cherouvim) [reply src/index.js]: Done.

**Review by cherouvim — COMMENTED** [review]
Requested changes done.

**Review by jmike — APPROVED** [review]

**Comment by jmike** [comment]
Accepted and merged manually.

**Pull request closed without merging**